changelog.d/app-2313.added

```diff
@@ -0,0 +1,3 @@
+returntocorp/semgrep-interfaces will now release a rule schema file in both yaml
```

Comment:
I am not sure this should go into the changelog, the changelog is for user-facing changes and this is an internal change.

Reply:
Removed